source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -377,11 +383,20 @@
                                 }
                             }
                         }
+                        valuesByItem.hasComments = ko.computed(function () {
+                            return _.some(valuesByItem.comments, function (comment) {
+                                return comment.HasComment() && comment.IsCalificationShown();
```

Comment:
I think that `hasComments` name looks confusing when it depends on _comments visibility_ not only on _comments existence_